packages/keyboard-direction-mixin/test/keyboard-direction-mixin.test.ts

```diff
@@ -6,8 +6,8 @@ import { KeyboardDirectionMixin } from '../keyboard-direction-mixin';
 
 const { expect } = chai;
 
-@customElement('kdm-element')
-class KdmElement extends KeyboardDirectionMixin(SlottedItemsMixin(LitElement)) {
+@customElement('kdm-element-vertical')
```

Comment:
Let's rename to `kdm-vertical-element` to follow the covention used in tests (end with `-element`).

Reply:
Done